dietpi/dietpi-software

```diff
@@ -6928,6 +6946,17 @@ exec sudo -u $ha_user dash -c '$ha_pyenv_activation; exec pip3 install -U homeas
 			fi
 
 		fi
+    
+		software_id=182 # Unbound
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 ))
+		then
+			Banner_Installing
+
+	    	G_AGI unbound
+
+			G_DIETPI-NOTIFY  2 'It is possible that Unbound just gave an error. This is safe to ignore.'
```

Comment:
Indeed it fails first but succeeds afterwards 🤔. I'll try to find out why this is, probably we find a fix.